### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by carawarner**
## Summary

Fixes https://refinery29.atlassian.net/browse/HD-2642

PR documents testing steps identified while working on https://refinery29.atlassian.net/browse/HD-2614.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,10 +0,0 @@
-YasOpenStack
```
Comment (schlueter) [inline README.rst]: rst format is required (afaik) for pypi.

Comment (carawarner) [inline README.rst]: Reformatting now.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+```
+
+## Making Changes
+
+
+First clone [yas](https://github.com/refinery29/yas), [YasOpenstack](https://github.com/refinery29/YasOpenstack), and [YasExampleHandlers](https://github.com/schlueter/YasExampleHandlers). That done, you can make your changes locally. To introduce a new command, create a handler. To alter an existing command, find and edit its handler.
+
+
+**Create a new Slackbot**
+_Don't test with the @openstack bot directly. You could disrupt development._
```
Comment (carawarner) [inline README.md]: Fixed.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+1. Go to https://refinery29.slack.com/apps/manage/custom-integrations
```
Comment (carawarner) [inline README.md]: Fixed.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+2. Select `Bots` and `Add Configuration`.
+3. Give your bot a name and click `Create`.
+4. Make note of the bot's name and its auto-generated API Token. You'll need these later.
+
+
+
+**Deploy your test Slackbot**
+1. Copy three files from `bot.cloud.rf29.net` to your local vagrant directory.
```
Comment (carawarner) [inline README.md]: Noted.

**Pull request merged**